genericx86-64-ext.coffee

```diff
@@ -0,0 +1,58 @@
+deviceTypesCommon = require '@resin.io/device-types/common'
+{ networkOptions, commonImg, instructions } = deviceTypesCommon
+
+GENERIC_FLASH = '''
+	Please make sure you do not have any other USB keys inserted.
+	Power up the <%= TYPE_NAME %>. Make sure you have a keyboard connected.
+	Press the F10 key (may differ on some platforms) while BIOS is loading in order to enter the boot menu.
+	Next, select the name of your USB key.
+'''
+
+GENERIC_POWERON = 'Power on your device.'
+
+postProvisioningInstructions = [
+	instructions.BOARD_SHUTDOWN
+	instructions.REMOVE_INSTALL_MEDIA
+	GENERIC_POWERON
+]
+
+module.exports =
+	version: 1
+	slug: 'genericx86-64-ext'
+	aliases: [ 'genericx86-64-ext' ]
+	name: 'Generic x86_64'
+	arch: 'amd64'
+	state: 'experimental'
```

Comment:
Let's put this straight to `released`. Or if not, to `new`. We do not use `experimental` anymore.

Reply:
Fixe, thanks.